Lib/sysconfig.py

```diff
@@ -172,7 +173,11 @@ def joinuser(*args):
 _BASE_PREFIX = os.path.normpath(sys.base_prefix)
 _EXEC_PREFIX = os.path.normpath(sys.exec_prefix)
 _BASE_EXEC_PREFIX = os.path.normpath(sys.base_exec_prefix)
+# Mutex guarding initialization of _CONFIG_VARS.
+_CONFIG_VARS_LOCK = threading.RLock();
 _CONFIG_VARS = None
+# True iff _CONFIG_VARS has been fully initialized.
```

Comment:
typo
```suggestion
# True if _CONFIG_VARS has been fully initialized.
```

Reply:
not a typo. if and only if

Reply:
Ah :sweat_smile:, I wasn't aware of that abbreviation. Thanks!